kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -219,8 +223,16 @@
         updateSection,
         handleReplacement,
         replacements,
+        allSections,
       } = injectQuizCreation();
 
+      const activeSectionTitle = computed(() => {
+        const activeSectionIndex = allSections.value.findIndex(section =>
+          isEqual(JSON.stringify(section), JSON.stringify(activeSection.value))
+        );
```

Comment:
There have been a couple places like this where I have to work out where in the list of sections the "current" section is and it makes me miss having a unique ID on them.

However, since sections cannot have duplicate questions between them AND a user cannot replace questions until they've added questions... this ought to do just fine unless JSON.stringify isn't as deterministic as I think.